roles/public_facing/tasks/download.ceph.com.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Put logrotate config in place
+  copy:
+    src: download.ceph.com/logrotate
+    dest: /etc/logrotate.d/download.ceph.com
+
+# Used for pushing upstream builds
+# https://github.com/ceph/ceph-build/blob/master/scripts/sync-push
+- name: Add signer user
+  user:
+    name: signer
+
+# signer_pubkey defined in inventory host_vars
+- name: Update signer user's authorized_keys
+  authorized_key:
+    user: signer
+    state: present
+    key: "{{ signer_pubkey }}"
+
+# Used to rsync download.ceph.com http logs and compile metrics
+# for metrics.ceph.com
+- name: Create Bitergia user
+  user:
+    name: bitergia
+    groups: adm
+
+# bitergia_pubkey defined in inventory host_vars
+- name: Update bitergia user's authorized_keys
+  authorized_key:
+    user: bitergia
+    state: present
+    key: "{{ bitergia_pubkey }}"
+
+- name: Put rrsync script in place for bitergia user
+  copy:
+    src: download.ceph.com/rrsync
+    dest: /home/bitergia/bin/rrsync
+    owner: bitergia
+    mode: 0774
+
+# Updates download.ceph.com/timestamp
+- name: Put make_timestamp script in place
+  copy:
+    src: download.ceph.com/make_timestamp
+    dest: /var/tmp/make_timestamp
```

Comment:
/var/tmp/ is also writable by all users, so that's a potential security issue (albeit unlikely to be one in practice). And for /usr/libexec, that's some old historic deviation from FHS from RH.